### PR TITLE
Controller changes

### PR DESCRIPTION
**Pull request by Ranagol**
Make FE and BE working for customer creation and customer update.

### CODE REVIEW

File: database/seeders/TmsInvoiceStatusSeeder.php
Comment (chrWZ) [inline database/seeders/TmsInvoiceStatusSeeder.php:1]: think thats not the solution. In the migration the name of the column was changed, so it need to be changed here as well. See branch dev_cwz_quickfixes

File: database/seeders/TmsOrderStatusSeeder.php
Comment (chrWZ) [inline database/seeders/TmsOrderStatusSeeder.php:1]: Same here:
See my branch dev_cwz_quckfixes

Comment (Ranagol) [reply database/seeders/TmsOrderStatusSeeder.php:1]: Understood. This issue is corrected in another branch, probably reviewed by Patrick. How should I proceed here, can I merge this? Or? @chrWZ

Comment (schoenenbergerdean) [reply database/seeders/TmsOrderStatusSeeder.php:1]: From my side this can be merged

**Review by schoenenbergerdean — APPROVED** [review]

**Pull request merged**